tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Packages/CreatedPackages.spec.ts

```diff
@@ -0,0 +1,385 @@
+    await umbracoApi.documentType.ensureNameNotExists(documentTypeName);
+    await umbracoApi.package.createEmptyPackage(packageName);
+    const documentTypeId = await umbracoApi.documentType.createDefaultDocumentTypeWithAllowAsRoot(documentTypeName);
+    const documentId = await umbracoApi.document.createDefaultDocument(documentName, documentTypeId);
+    await umbracoUi.reloadPage();
+
+    // Act
+    await umbracoUi.package.clickExistingPackageName(packageName);
+    // The frontend has updated the button name to "Choose" of "Add". But they are a bit unsure if they want to change it to select instead.
+    // So for the moment I have used the page instead of our UiHelper. Because it is easier to change the locator.
+    // await umbracoUi.package.clickAddContentToPackageButton();
+    await page.locator('[label="Content"] >> [label="Choose"]').click();
+    await umbracoUi.package.clickLabelWithName(documentName);
+    await umbracoUi.package.clickChooseBtn();
+    await umbracoUi.package.clickSaveChangesToPackageButton();
+
+    // Assert
+    const packageData = await umbracoApi.package.getByName(packageName);
+    expect(packageData.contentNodeId == documentId).toBeTruthy();
+    await umbracoUi.package.clickExistingPackageName(packageName);
+    expect(umbracoUi.package.isButtonWithNameVisible(documentName + ' ' + documentId)).toBeTruthy();
+
+    // Clean
+    await umbracoApi.documentType.ensureNameNotExists(documentTypeName);
+    await umbracoApi.package.ensureNameNotExists(packageName);
+  });
+
+  // Currently unable to run this test. Because you are not able to save a mediaId
+  test.skip('can create a package with media', async ({umbracoApi, umbracoUi}) => {
+    // Arrange
+    const mediaTypeName = 'TestMediaType';
+    const mediaName = 'TestMedia';
+    await umbracoApi.mediaType.ensureNameNotExists(mediaTypeName);
+    await umbracoApi.package.createEmptyPackage(packageName);
+    const mediaTypeId = await umbracoApi.mediaType.createDefaultMediaType(mediaTypeName);
+    const mediaId = await umbracoApi.media.createDefaultMedia(mediaName, mediaTypeId);
+    await umbracoUi.reloadPage();
+
+    // Act
+    await umbracoUi.package.clickExistingPackageName(packageName);
+    await umbracoUi.package.clickAddMediaToPackageButton();
+    await umbracoUi.package.clickCaretButton();
+    await umbracoUi.package.clickLabelWithName(mediaName);
+    await umbracoUi.package.clickSubmitButton();
+    await umbracoUi.package.clickSaveChangesToPackageButton();
+
+
+    // Assert
+    await umbracoUi.package.clickExistingPackageName(packageName);
+    expect(umbracoUi.package.isButtonWithNameVisible(mediaTypeName + ' ' + mediaId)).toBeTruthy();
+    const packageData = await umbracoApi.package.getByName(packageName);
+    expect(packageData.mediaIds[0] == mediaId).toBeTruthy();
+
+    // Clean
+    await umbracoApi.mediaType.ensureNameNotExists(mediaTypeName);
+    await umbracoApi.package.ensureNameNotExists(packageName);
```

Comment:
Should remove "await umbracoApi.package.ensureNameNotExists(packageName);" in Clean section of all tests as it is already to be called in afterEach method